### PR TITLE
AC Test Fix - LRAC-10446 | master

### PR DESCRIPTION
**Pull request by ViniLopes20**
https://issues.liferay.com/browse/LRAC-10446
https://testray.liferay.com/home/-/testray/case_results?testrayBuildId=1602600055&testrayRunId=1602608396

### CODE REVIEW

**Comment by ViniLopes20** [comment]
ci:test:sf

**Comment by ViniLopes20** [comment]
ci:test:sf

**Review by AfonsoHSantos — APPROVED** [review]

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/CustomEventsBlockEvents.testcase [hunk 1]
```diff
@@ -368,9 +368,15 @@ definition {
 			ACEventAnalysis.viewAnalysisInformation(informationValueList = "1");
 		}
 
-		task ("Navigation to custom events tab") {
+		task ("Navigation to settings and discard analysis change") {
```
Comment (alexcandidos) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/CustomEventsBlockEvents.testcase:371]: hey @ViniLopes20 , what do you think if we update this task saying:
`Navigation to settings and discard analysis change if needed`

Comment (ViniLopes20) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/CustomEventsBlockEvents.testcase:371]: Yes, @alexcandidos. Sounds good!

**Review by manuferreira — APPROVED** [review]

**Comment by alexcandidos** [comment]
Hi @ViniLopes20 , Thanks for working on it!

The code looks good to me!

I'm forwarding it.

**Comment by alexcandidos** [comment]
ci:forward

**Review by alexcandidos — APPROVED** [review]

**Pull request closed without merging**